Sources/web3swift/Utils/Extensions/Data+Extension.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  Data+Extension.swift
+//
+//  Created by JeneaVranceanu on 18.10.2023.
+//
+
+import Foundation
+
+extension Data {
+
+    func asJsonDictionary() throws -> [String: AnyObject]? {
+        try JSONSerialization.jsonObject(with: self, options: .mutableContainers) as? [String:AnyObject]
+    }
+}
```

Comment:
We're still haven't added SwiftLint ruleset here, and I wonder would we ever do this, but I believe that such a set that we're having in a stash contains the rule about keeping one empty line at the very end of each file.